apps/api/src/server/migration/migrators/nsip-project-migrator.js

```diff
@@ -54,20 +55,30 @@ export const migrateNsipProjects = async (models) => {
 
 		const eventType = model.publishStatus === 'published' ? EventType.Publish : EventType.Update;
 
-		await broadcastNsipProjectEvent(nsipProject, eventType);
+		if (model.migrationStatus) {
+			console.info(`Broadcasting event for case ${model.caseId}`);
```

Comment:
Ideally we use the pino logger rather than console - so we get proper JSON formatted logs.

Reply:
https://github.com/Planning-Inspectorate/back-office/pull/2409/commits/7efff32722d2f5f9900fd199edc3a2a71c637552